odoo/addons/base/models/res_users.py

```diff
@@ -1009,19 +1009,9 @@ def action_revoke_all_devices(self):
             'context': ctx,
         }
 
-    @api.model
     @api.readonly
+    @tools.ormcache('self.id', 'group_ext_id')
     def has_group(self, group_ext_id):
```

Comment:
Before this change, calling `has_group()` in XML-RPC could only provide information about the current user. After the change, one can call the method in XML-RPC to retrieve information about other users.